modules/actions/workflows.go

```diff
@@ -255,9 +255,17 @@ func matchPushEvent(commit *git.Commit, pushPayload *api.PushPayload, evt *jobpa
 				matchTimes++
 			}
 		case "paths":
-			filesChanged, err := commit.GetFilesChangedSinceCommit(pushPayload.Before)
+			var (
+				filesChanged []string
+				err          error
+			)
+			if pushPayload.Before != git.EmptySHA {
+				filesChanged, err = commit.GetFilesChangedSinceCommit(pushPayload.Before)
+			} else {
+				filesChanged, err = commit.GetFilesChanged()
+			}
```

Comment:
Why not reuse "GetFilesChangedBetween" and make it support EmptySHA ?

Reply:
done in b5c32d64dd62751917de58e3111018fe66190e38